FilePersistence/src/version_control/Commit.cpp

```diff
@@ -108,22 +108,23 @@ bool Commit::isValidMatch(const char* content, qint64 size, const char* indexOfI
 {
 	start = indexOfId-content;
 	end = start;
+	int precClosedBracketCnt = 0;
 	// start is the first character of the line containing id
 	while (start >= 0 && content[start] != '\n')
 	{
-		// String is of the form {.*} {id}
-		if (!findChildrenByParentId && content[start] == '}') return false;
+		// Remove fake-ids
+		if (content[start] == '}')
+		{
+			if (!findChildrenByParentId)	return false;
```

Comment:
Please merge this condition into the next one (so have only one if after the ++)